src/main/webapp/encounters/encounter.jsp

```diff
@@ -674,8 +674,8 @@ var encounterNumber = '<%=num%>';
 
 				//let's see if this user has ownership and can make edits
       			boolean isOwner = ServletUtilities.isUserAuthorizedForEncounter(enc, request,myShepherd);
-            boolean isPublic = ServletUtilities.isEncounterOwnedByPublic(enc) && request.getUserPrincipal() != null; //should at least be logged in
-      			pageContext.setAttribute("editable", (isOwner || isPublic) && CommonConfiguration.isCatalogEditable(context));
+            boolean isPublic = ServletUtilities.isEncounterOwnedByPublic(enc);
+            pageContext.setAttribute("editable", (isOwner) && CommonConfiguration.isCatalogEditable(context));
```

Comment:
for example, this `pageContext...` line should then use `(isOwner || isPublicEditable)`